test/e2e/host_path.go

```diff
@@ -53,9 +53,13 @@ var _ = Describe("hostPath", func() {
 	})
 
 	AfterEach(func() {
-		By(fmt.Sprintf("Destroying namespace for this suite %v", namespace.Name))
-		if err := deleteNS(c, namespace.Name, 5*time.Minute /* namespace deletion timeout */); err != nil {
-			Failf("Couldn't delete ns %s", err)
+		if testContext.DeleteNamespace {
+			By(fmt.Sprintf("Destroying namespace for this suite %v", namespace.Name))
+			if err := deleteNS(c, namespace.Name, 5*time.Minute /* namespace deletion timeout */); err != nil {
+				Failf("Couldn't delete ns %s", err)
+			}
+		} else {
+			Logf("Found DeleteNamespace=false, skipping namespace deletion!")
```

Comment:
I would like to avoid having this code in multiple places - it's good time to migrate those tests to framework before doing it.